bpf/lib/common.h

```diff
@@ -171,6 +171,7 @@ struct endpoint_info {
 struct remote_endpoint_info {
 	__u16		sec_label;
 	__u16		pad[3];
+	__u32		tunnel_endpoint;
```

Comment:
If I follow correctly, this is an optional IPv4 address of the node where this endpoint can be found. Might be useful to add a comment here to describe this.

It can also use the latter 32 bits of the pad field to avoid resizing the structure.